### PR TITLE
Update twin tests and fix some cases

### PR DESCRIPTION
**Pull request by A-Harby**
### Description

Fix failed tests from the workflow.

### Changes

- Update new twin TFT balance.
- Add new method to randomly pick a public IPV4.
- Update IPV6 with a correct public IPV6. 
- Update outdated xpath.
- Remove delete twin test case.
- Update twin with relay changes.
- Fix timeout in some test cases.
- decreasing number of new farms being created.

### Related issues

Some of the tests are still failing but they are all reported.
- https://github.com/threefoldtech/tfgrid_dashboard/issues/521
- https://github.com/threefoldtech/tfgrid_dashboard/issues/509
- https://github.com/threefoldtech/tfgrid_dashboard/issues/496
- https://github.com/threefoldtech/tfgrid_dashboard/issues/444

### Checklist

- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: tests/frontend_selenium/tests/test_farm.py [hunk 1]
```diff
@@ -218,61 +149,29 @@ def test_add_invalid_farmpayout_address(browser):
...
+    polka_page.authenticate_with_pass(password)
+    assert farm_page.wait_for('Transaction submitted')
+    assert farm_page.wait_for('Address added!')
+    assert farm_page.wait_for('GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4')
 
 
-def test_valid_ip(browser):
+def test_ip(browser):
     """
     Test Case: TC1141 - Enter valid IP
```
Comment (mohamedamer453) [inline tests/frontend_selenium/tests/test_farm.py:171]: The test case was remove but its title is still available and it replaced the title of the next case.

same issue is present in other functions in the same file.

File: tests/frontend_selenium/tests/test_twin.py [hunk 2]
```diff
@@ -22,119 +22,68 @@ def before_test_setup(browser, create_account):
...
-    assert twin_page.wait_for('Planetary using Yggdrasil IPV6')
-    assert 'Planetary using Yggdrasil IPV6' in browser.page_source
-    assert twin_page.Button_why_doIeven_need_twin(
-    ) == 'https://library.threefold.me/info/manual/#/manual__yggdrasil_client'
+    assert twin_page.wait_for('Accepted!')
+    assert twin_page.wait_for('Choose a Relay Address')
 
 
-def test_create_twin_IP(browser):
+def test_create_twin_relay(browser):
```
Comment (mohamedamer453) [inline tests/frontend_selenium/tests/test_twin.py:39]: 2 Tests in the test_twin file failed as seen below.

image

full logs can be found [here](https://gist.github.com/mohamedamer453/d319fbccf48605359283d57b0e56e6f8).

**Review by mohamedamer453 — CHANGES_REQUESTED** [review]
Other than the 2 comments that i mentioned, everything looks good and i tried all of the tests and everything worked without any issues :+1:

File: tests/frontend_selenium/tests/test_twin.py (hunk 2 above)
Comment (mohamedamer453) [reply tests/frontend_selenium/tests/test_twin.py:39]: The issue was resolved and the `edit_twin_relay` test doesn't fail anymore

File: tests/frontend_selenium/tests/test_farm.py [hunk 3]
```diff
@@ -218,61 +149,29 @@ def test_add_invalid_farmpayout_address(browser):
...
+    farm_page.setup_farmpayout_address(farm_name)
+    farm_page.add_farmpayout_address(case).click()
+    polka_page.authenticate_with_pass(password)
+    assert farm_page.wait_for('Transaction submitted')
+    assert farm_page.wait_for('Address added!')
+    assert farm_page.wait_for('GA2CWNBUHX7NZ3B5GR4I23FMU7VY5RPA77IUJTIXTTTGKYSKDSV6LUA4')
 
 
-def test_valid_ip(browser):
+def test_ip(browser):
```
Comment (mohamedamer453) [inline tests/frontend_selenium/tests/test_farm.py:169]: All ip related tests in the test_farm file are failing.

image

Full logs can be found [here](https://gist.github.com/mohamedamer453/10ea68cb1ecd6285bd73ec8d1d458e45).

**Review by mohamedamer453 — CHANGES_REQUESTED** [review]
The mentioned issues in the last review were fixed. However new issues appeared in  `test_farm`,  other than that everything else is running smoothly and tested multiple times without any issues.

File: tests/frontend_selenium/tests/test_farm.py (hunk 3 above)
Comment (A-Harby) [reply tests/frontend_selenium/tests/test_farm.py:169]: It's an issue in the farm page, adding IP to the farm.
And its reported and once its fixed the tests should pass.
https://github.com/threefoldtech/tfgrid_dashboard/issues/552

**Review by mohamedamer453 — APPROVED** [review]
The issues related to the `test_farm` page wasn't related to the tests #552

**Pull request merged**